config/repositories/tag_repos.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Hi Nguyen,
I'm currently not clear about the tagging tool for the OSS-version.
I think we will use - as we do now - gitlab to trigger a build, then tagging tool will tag also RobotFramework_AIO repository accordingly, which then triggers the corresponding OSS build.
Thank you,
 Thomas
Thank you,
 Thomas

Reply:
I deleted unsued tag_repos.json from github